index.html

```diff
@@ -69,7 +69,7 @@
   <script src="https://cdn.jsdelivr.net/npm/leaflet-providers@2.0.0/leaflet-providers.min.js" integrity="sha256-MoLxVCCJOBvyaZFUhQu9NK/gdYqtyuZgKwjAajrC5wc=" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
   <script src="https://cdn.jsdelivr.net/npm/leaflet-omnivore@0.3.4/leaflet-omnivore.min.js" integrity="sha256-k1l+ouvMXT+0iRA8pc8e0dge4ZeGjXG6imrvWboFTRE=" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
   <script src="https://cdn.jsdelivr.net/npm/leaflet.locatecontrol@0.79.0/dist/L.Control.Locate.min.js" integrity="sha256-jVdNHjjOOJMoykxLOdGxOUzGJDlmr8MM6sFF++b1/sI=" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
-  <script src="https://cdn.jsdelivr.net/npm/leaflet-touch-helper@2.0.0/leaflet-touch-helper.min.js" integrity="sha256-9DZC2FEK/x85mjAOJcI6WeI7m7w9zLnmV1pTKrCFc7U=" crossorigin="anonymous" referrerpolicy="no-referrer"></script>

```

Comment:
@Sp3EdeR minimization caused a load issue?

Reply:
No, the SRI was its problem. It was not provided by jsdelivr and I generated it with a different tool, but it was not accepted by some mobile browsers. So I just took the most used file from jsdelivr as-is, which is the non-minimized one.